pkg/gather/service/analyze.go

```diff
@@ -85,6 +85,7 @@ func analyzeGatherBundle(bundleFile io.Reader) error {
 		optional bool
 	}{
 		{name: "release-image", check: checkReleaseImageDownload, optional: false},
+		{name: "node-image-pull", check: checkNodeImagePull, optional: false},
 		{name: "bootkube", check: checkBootkubeService, optional: false},
```

Comment:
```suggestion
        {name: "node-image-pull", check: checkNodeImagePull, optional: false},
		{name: "release-image", check: checkReleaseImageDownload, optional: false},
		{name: "bootkube", check: checkBootkubeService, optional: false},
```

I think the order matters right, according to https://github.com/openshift/installer/pull/4751#discussion_r607331794? IIUC, the `node-image-pull` is first to start before the other two :thinking: (the `release-image` is never started when `node-image-pull` is throwing errors...